plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -1508,10 +1528,20 @@ public void registerCoreMembers() {
         // @Description
         // The flag command sets or modifies custom value storage database entries connected to
         // each player, each NPC, each entity, and the server.
-        // All the flag values are stored default in "plugins/denizen/saves.yml" file.
+        // Flags can have operations performed upon them, such as:
+        // Increment a flag by 1:
+        // - flag player counter:++
+        // Increment a flag by 3:
+        // - flag player counter:+:3
+        // Decrement a flag by 2:
+        // - flag player counter:-:2

```

Comment:
Increment and decrement are _not_ the same words as add and subtract. "Increment" _means definitively_ that it goes up by one.

Reply:
mergu gave that example haha, I'll change it.